src/state/dashboard/reducer.js

```diff
@@ -87,8 +87,8 @@ export function getJobsByStatus(state) {
 }
 
 function convertSecToMinHours(sec) {
-  const hours = Math.floor(sec / 3600),
-    minutes = Math.floor((sec % 3600) / 60);
+  const hours = Math.round(sec / 3600),
```

Comment:
For hours, you actually don't want to round it because we only care about the integer part.

Reply:
Only the value in the finest unit should be rounded.